src/Emanote/View/Template.hs

```diff
@@ -93,8 +93,10 @@ renderLmlHtml emaAction model note = do
       pageTitle = M.modelLookupTitle r model
       noteRenderers =
         NoteRenderers
-          [PF.urlResolvingSplice]
-          [ PF.embedWikiLinkResolvingSplice,
+          [ PF.embedInlineWikiLinkResolvingSplice,
+            PF.urlResolvingSplice
```

Comment:
Is there a reason for changing the order of `urlResolvingSplice` (it is no longer the first spice in this PR)?

Reply:
This was because having `urlResolvingSplice` first does not recognize the inline element (Link) as image but as File embed but not rendered as image. Putting first `PF.embedInlineWikiLinkResolvingSplice` fixed that.

Reply:
In that case, we should add a code comment saying exactly that (for posterity).